### PR TITLE
add Technology and Project scheduling

### PR DESCRIPTION
**Pull request by SanjulaGanepola**

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -53,23 +53,68 @@ \section{Proof of Concept Demonstration Plan}
...
+
+\subsubsection{Continuous Integration Tools}
+- GitHub actions
+- testing workflow
+- code coverage workflow
+- build workflow
+- docs workflow
+
+\subsubsection{Performance Tools}
+TODO: What to put here?
```
Comment (SanjulaGanepola) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:87]: Selenium

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -53,23 +53,68 @@ \section{Proof of Concept Demonstration Plan}
...
+
+\subsubsection{Libraries}
+- React Native Elements: UI toolkit/component library
+- VisionCamera for React Native: Camera related features
+
```
Comment (SanjulaGanepola) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:92]: - Encryption/decryption
- database access

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 3]
```diff
@@ -53,23 +53,68 @@ \section{Proof of Concept Demonstration Plan}
...
 
 \section{Coding Standard}
 
 \section{Project Scheduling}
-
-\wss{How will the project be scheduled?}
+- Add course outline deliverable deadlines
+- github projects
+
+TODO: Add dates for major development milestones:
```
Comment (SanjulaGanepola) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:110]: - Connect app to database

**Review by CaKrome — APPROVED** [review]
Looks good

**Pull request merged**